src/lib/shared/Tag.svelte

```diff
@@ -1,25 +0,0 @@
-<script lang="ts">
-	export let size: string = 'small';
-	export let style: string | undefined = undefined;
-	export let color: string = 'pink';
-</script>
-
-<span
-	class={`tag ${size === 'large' ? 'typo-regular' : 'typo-text-small'}`}
-	class:small={size === 'small'}
-	class:large={size === 'large'}
-	style={`color: var(--color-${color}); background-color: var(--color-${color}-darker); ${style}`}
-	><slot /></span
->
-
-<style>
-	.tag {
-		padding: 0.1875rem 0.5rem 0.3125rem;
-		border-radius: 0.75rem;
-	}
-
-	.large {
-		padding: 0.1875rem 0.75rem 0.3125rem;
-		border-radius: 1rem;
-	}
-</style>
```

Comment:
Why delete it? If you're not useing it anywhere. Since we want to bring it back

Reply:
Good point, I did not want to delete it completely but was expecting to create a new component based on some work I have done previously. I will add it again, though we still need to figure out how to resolve the errors / warnings that this component introduces.

Reply:
Looks like the error does not appear anymore. I added the component again.